templates/list_pages/node--oe-list-page--full.html.twig

```diff
@@ -1,5 +1,5 @@
 {% apply spaceless %}
-  {{ content|without('extra_field_oe_list_page_filters:node:oe_list_page', 'extra_field_oe_list_page_results:node:oe_list_page', 'extra_field_oe_list_page_selected_filters:node:oe_list_page', 'extra_field_oe_list_page_total_count:node:oe_list_page', 'extra_field_oe_list_page_sort:node:oe_list_page') }}
+  {{ content|without('extra_field_oe_list_page_filters:node:oe_list_page', 'extra_field_oe_list_page_results:node:oe_list_page', 'extra_field_oe_list_page_selected_filters:node:oe_list_page', 'extra_field_oe_list_page_total_count:node:oe_list_page', 'extra_field_oe_list_page_sort:node:oe_list_page', 'oe_summary', 'oe_featured_media') }}
```

Comment:
These should be hidden for the full view in the UI, the configs are probably broken (I commented them too)